### PR TITLE
Rename fixes/services

### PR DESCRIPTION
**Pull request by Jonathan1600**
### Description

**What work was done?**
- Fixed and renamed nav bar to accommodate new naming credentials for table schemas
- Fixed and renamed My Profile page to accommodate new naming credentials for table schemas
- Fixed and renamed Programs Page to accommodate new naming credentials for table schemas

**Why was this work done?**
- Necesary for basic functionality

### CODE REVIEW

File: src/components/common/ProgramsTable/ProgramTable.js [hunk 1]
```diff
@@ -33,9 +33,9 @@ const ProgramTable = ({
 
   const edit = record => {
     form.setFieldsValue({
-      name: '',
-      type: '',
-      description: '',
+      program_name: '',
```
Comment (Monicascz) [inline src/components/common/ProgramsTable/ProgramTable.js:36]: thanks for your diligent effort in going through each one of these fields to rename after our backend team rebuilt the table schema.

**Review by Monicascz — APPROVED** [review]
Lots of work to still go through to get the rest of our code rendering, but really solid start! Thanks for taking this on!

**Review by robertjallen — APPROVED** [review]

File: src/state/actions/ServiceTypeActions.js [hunk 2]
```diff
@@ -32,6 +32,7 @@ export const getAllServiceTypesAction = () => dispatch => {
     .get('/api/service_types')
     .then(res => {
       dispatch({ type: GET_ALL_SERVICE_TYPES_SUCCESS, payload: res.data });
+      console.log(res.data);
```
Comment (PurdyRad) [inline src/state/actions/ServiceTypeActions.js:35]: Probably useful for us in development to see what we're getting back, but definitely wanna remove it before we're finished and ready for production!

**Review by PurdyRad — APPROVED** [review]
Overall awesome work, saw one console.log that we could remove but for now it may be useful for us to see that data when it comes in. Great job on matching up the new variable names though, definitely something we had to get done!

**Pull request merged**